pkg/config/config.go

```diff
@@ -163,8 +166,8 @@ type Config struct {
 			Token           string
 			SupportsInvites bool `mapstructure:"supports_invites"`
 		}
-		LDAP *LDAP
-		OIDC OIDC
+		RemoteAuthenticator *RemoteAuthenticator `mapstructure:"remote_authenticator"`
+		OIDC                OIDC                 `mapstructure:"oidc"`
```

Comment:
is this relevant?

Reply:
Yes, The OIDC itself config is relevant. (From previous PR's) 
here it's only ``mapstructure:"oidc"`` following comment previously about alignment incase field name change.